packages/ckeditor5-basic-styles/src/bold/boldui.ts

```diff
@@ -55,4 +55,20 @@ export default class BoldUI extends Plugin {
 			return view;
 		} );
 	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public afterInit(): void {
```

Comment:
I'm not a fan of this approach of registering new keystrokes. I remember suggesting the use of a static field that could be utilized by `AccessibilityDialog` to minimize the 'connections' between plugins. Anyway, we can still proceed with this solution.

@scofalik, do you have any preferences?

Reply:
For the record, whatever solution we choose the API of the plugin (methods) must remain in case sb. wants to register some new entries on the fly (e.g. via some external integration).

Reply:
Each time I return to this subject, I feel that gathering a11y data should be separated from displaying it. I wish so much it was possible to automate it somehow but I guess this not possible currently, without a broad refactor in many many places.

I propose two solutions:

1.  Something like `editor.accessibilityHelp` where we would add registering methods. Then, `AccessibiltyDialog` will read from it and plugins will add to it. It will be always available and we will decouple plugins from the a11y dialog plugin.
2.  Put this functionality into `editor.keystrokes`. We could have something like `editor.keystroke.declare()` that would not actually create a keystroke but would save the information about it. We could have partial automation. We could read from the keystrokes that already exist, and where needed we would add `declare()`. This solution is better if we want to semi-automate it, but it is far less straightforward than `editor.accessibilityHelp.registerKeystroke()`. Especially if we will go deeper with accessibility help and we will start adding some more data there, and it will be clear that "accessibility help" is its own "data space" not just a list of keystrokes.